src/topology/metric_space/hausdorff_distance.lean

```diff
@@ -1023,6 +1034,37 @@ by simp_rw [thickening, inf_edist_closure]
 @[simp] lemma cthickening_closure : cthickening δ (closure s) = cthickening δ s :=
 by simp_rw [cthickening, inf_edist_closure]
 
+open ennreal
+
+lemma _root_.disjoint.exists_thickenings (hs : is_compact s) (ht : is_closed t)
+  (hst : disjoint s t) :
```

Comment:
Could you put the `disjoint s t` assumption as the first assumption, to make sure dot notation works optimally?